packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,412 @@
-  &::-webkit-search-results-decoration {
-    appearance: none;
-  }
-
-  &::placeholder {
-    color: #5a5e9a;
-  }
-
-  &:focus {
-    border-color: #3c4fe0;
-    box-shadow: rgb(35 38 59 / 5%) 0 1px 0 0;
-    outline: currentColor none medium;
-  }
-}
-
-.aa-Label[hidden],
-.aa-LoadingIndicator[hidden] {
-  display: none;
-}
-
-.aa-SubmitButton,
-.aa-ResetButton,
-.aa-LoadingIndicator,
-.aa-TouchSearchButtonIcon {
-  height: 100%;
-  padding: 0 0.5rem;
-}
-
-.aa-SubmitButton,
-.aa-ResetButton {
-  background: none;
-  border: 0;
-  color: inherit;
-  cursor: pointer;
-}
-
-.aa-LoadingIndicator {
-  position: absolute;
-  z-index: 2;
-}
-
-.aa-LoadingIcon {
-  height: 100%;
-}
-
-.aa-List {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-.aa-Item {
-  color: #23263b;
-  cursor: pointer;
-  display: flex;
-  grid-gap: 0.5rem;
-  justify-content: space-between;
-
-  mark {
-    background: none;
-    font-style: normal;
-    font-weight: bold;
-  }
-
-  /* stylelint-disable-next-line */
-  &[aria-selected='true'] {
-    background-color: #f5f5fa;
-  }
-}
-
-.aa-ItemLink {
-  color: inherit;
-  text-decoration: none;
-}
-
-.aa-ItemContent {
-  display: flex;
-  flex-grow: 1;
-  grid-gap: 0.5rem;
-  padding: 0.5rem;
-}
-
-.aa-ItemSourceIcon,
-.aa-ItemActionButton {
-  color: rgba(0, 0, 0, 0.4);
-}
-
-.aa-ItemActionButton {
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  opacity: 0.8;
-  padding: 0.5rem;
-  transition: color 100ms;
-  @media screen and (prefers-reduced-motion: reduce) {
-    transition: none;
-  }
-
-  &:hover,
-  &:focus {
-    color: rgba(80, 80, 80, 1);
-  }
+// ----------------
+// Darkmode
+// ----------------
+body {
+    &[data-theme='dark'],
+    &.dark {
+        --primary-color: rgb(247, 247, 0);
+        --muted-color: rgba(167, 167, 96, 0.6);
+        --selected-color: rgba(247, 247, 0, 0.1);
+        --icon-color: rgb(31, 207, 248);
+        --text-color: rgb(183, 192, 199);
+        --background-color: rgb(21, 23, 42);
+        --panel-shadow: inset 1px 1px 0 0 rgb(44, 46, 64), 0 3px 8px 0 rgb(0, 3, 9);
+    }
 }
 
-.aa-TouchOverlay {
-  backdrop-filter: blur(16px);
-  background: rgba(255, 255, 255, 0.95);
-  bottom: 0;
-  left: 0;
-  padding: 0.5rem;
-  position: fixed;
-  right: 0;
-  top: 0;
+// ----------------
+// Autocomplete
+// ----------------
+.aa-Autocomplete {
+    font-size: var(--font-size);
+    text-align: left;
+    line-height: 1em;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    // searchbox
+    form.aa-Form {
+        position: relative;
+        display: flex;
+        border: 1px solid var(--muted-color);
+        align-items: center;
+        padding: 0 var(--spacing) 0 var(--spacing-half);
+        line-height: 1em;
+        background-color: var(--background-color);
+        border-radius: 3px;
+        &,
```

Comment:
```suggestion
```

(this line makes the following styles apply always I think)

Reply:
weird line, good catch